### PR TITLE
fix(issue#261): Fixed Forbidden error template

### PR DESCRIPTION
**Pull request by alexbojko**
Fixed Forbidden error template instead of raise Key error `KeyError: 'user not in this class'`

@z4y4ts please review

### CODE REVIEW

File: mysite/ct/views.py [hunk 1]
```diff
@@ -329,7 +329,14 @@ def course_view(request, course_id):
     'show courselets in a course'
     course = get_object_or_404(Course, pk=course_id)
     if is_teacher_url(request.path):
-        notInstructor = check_instructor_auth(course, request)
+        try:
```
Comment (maxsocl) [inline mysite/ct/views.py]: Eventually, I do not like this repeatable code. Need generalize somehow.

File: mysite/ct/tests/views.py [hunk 2]
```diff
@@ -683,6 +683,16 @@ def test_edit_unit_not_instructor(self):
...
         self.assertRedirects(response, reverse('ct:study_unit', args=(self.course.id, self.unit.id)))
 
+
+    def test_edit_unit_no_role(self):
+        self.role.delete()
+        response = self.client.get(
+            reverse('ct:edit_unit', kwargs={'course_id': self.course.id, 'unit_id': self.unit.id})
+        )
+        self.assertEqual(response.status_code, 302)
+        self.assertRedirects(response, reverse('ct:study_unit', args=(self.course.id, self.unit.id)))
```
Comment (z4y4ts) [inline mysite/ct/tests/views.py]: `assertRedirects` by default checks if `response.status_code` is `302`

File: mysite/ct/tests/views.py [hunk 3]
```diff
@@ -453,8 +453,8 @@ def test_course_view_no_role(self):
         self.course = Course(title='test_title', addedBy=self.user)
         self.course.save()
         self.client.login(username='test', password='test')
-        with self.assertRaises(KeyError):
-            self.client.get(reverse('ct:course', kwargs={'course_id': self.course.id}))
+        response = self.client.get(reverse('ct:course', kwargs={'course_id': self.course.id}))
+        self.assertEqual(response.status_code, 302)
```
Comment (z4y4ts) [inline mysite/ct/tests/views.py]: What about to check where user is being redirected?

File: mysite/ct/views.py [hunk 4]
```diff
@@ -1067,7 +1079,15 @@ def edit_lesson(request, course_id, unit_id, ul_id):
...
     course = get_object_or_404(Course, pk=course_id)
-    notInstructor = check_instructor_auth(course, request)
+    try:
+        notInstructor = check_instructor_auth(course, request)
+    except KeyError as e:
+        return render(
+            request,
+            'lti/error.html',
+            {'message': 'This action is not allowed for this user ({})'.format(e.message)},
+            status=404
```
Comment (maxsocl) [inline mysite/ct/views.py]: @alexbojko this is wrong status code for this situation. We need 403.

File: mysite/ct/views.py (hunk 1 above)
Comment (maxsocl) [inline mysite/ct/views.py]: @alexbojko Again. This is the same repetition. I want to suggest you change `check_instructor_auth` to simply return Boolean value. For this please review all places where this function is used. Thanks.

**Pull request merged**